app/views/projects/index.html.erb

```diff
@@ -72,10 +72,6 @@
               apply_project_applications_path(id: project.id), method: :post, class: 'btn btn-primary btn-xs' %>
           <% end %>
         <% end %>
-        <% if can? :leave_project, project %>
-            <%= link_to t('.leave_project', default:'Leave Project'),
-                        sign_user_out_project_path(user_id: current_user.id, id: project), method: :delete, class: 'btn btn-default btn-xs' %>
-        <% end %>
```

Comment:
Ah. Das hab ich nciht gesehen. Das kommt noch durch komische Merge-Dinge. Ich denke das ist dann bei allen Project-Datein und in der Ability.rb so?